manifest.json

```diff
@@ -7,9 +7,17 @@
       "js": [ "mustache.min.js", "content.js" ],
       "matches": [ "*://*/health", "*://*/__health", "*://*/*/health", "*://*/*/__health", "*://*/*/health?*", "*://*/*/__health?*" ],
       "run_at": "document_end"
-   } ],
+   }],
    "icons": {
       "128": "icon_128.png"
    },
-   "web_accessible_resources": [ "template.html", "format.js" ]
+   "web_accessible_resources": [ "template.html", "format.js" ],
+    "background": {
+    "scripts": ["json_request.js"]
+   },
+   "permissions": [
+          "webRequest",
+	  "webRequestBlocking",
+          "*://*.ft.com/__health"
```

Comment:
Given the server will run the content negotiation process the pattern  "*://*/__health" would not be over-broad. However getting this right for *ft.com has immediate independent value to our team.